tools/rdm/TestDefinitions.py

```diff
@@ -680,6 +680,17 @@ class GetSupportedParameters(ResponderTestFixture):
       ('LOCK_STATE', 'LOCK_STATE_DESCRIPTION'),
   ]
 
+  # If any PID apart from the first in each group is supported, the first one
+  # must be.
+  PID_REVERSE_DEPENDENCIES = [
```

Comment:
I'd make this a list of tuples in the form:

[  
  ( first_pid, [other_pids]),
]